test/producer-consumer-spec.js

```diff
@@ -141,4 +141,17 @@ describe('producer/consumer', function () {
       });
     });
   });
+
+  describe('undefined queue name', () => {
+    before(() => {
+      bunnymq.connection._config.consumerSuffix = undefined;
+    });
+
+    it('should receive message', () => {
+      bunnymq.consume('queue-name-undefined-suffix', (message) => {
+        assert(message.msg === 'test for undefined queue suffix');
```

Comment:
I guess he means it's atypical usage. usually  [assert.equal](https://nodejs.org/api/assert.html#assert_assert_equal_actual_expected_message) is used when comparing 2 values.